packages/react-native-bpk-component-rating/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "react-native-bpk-component-rating",
+  "version": "0.0.1",
+  "main": "index.js",
+  "description": "Backpack React Native rating component.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": "^16.8.3",
+    "react-native": ">= 0.59.0"
+  },
+  "dependencies": {
+    "bpk-tokens": "^28.0.1",
+    "prop-types": "^15.5.8"
+  },
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
```

Comment:
Is this a new thing? Haven't seen it before

Reply:
not sure, I've copied it from another package so I guess is been there for a while